README.md

```diff
@@ -1,57 +1,67 @@
+- [🐞 Report a Bug][1]
+- [❓ Ask a question][2]
+- [🙋 Start contributing][3]
 
 
 ## Topo: The CabinDAO Design System
 
 Welcome to Topo, CabinDAO's internal design system. We use this collection of
-libraries and tools to build high-quality interfaces.
-
-Topo consists of two libraries:
-
-1. [Topo Theme](./packages/theme/README.md)
-1. [Topo Components](./packages/components/README.md)
+libraries and tools to build high-quality interfaces while maintaining brand
+consistency and uniformity.
 
-The [Topo Theme](./packages/theme/README.md) is built on top of
-[Stitches](https://stitches.dev/docs/tokens) and includes all the primitives for
-the CabinDAO visual identity. [Topo Components](./packages/components/README.md)
-is our library of pre-built UI components.
+The Topo theming is built on top of [Stitches][4] and includes all the primitives 
+for the CabinDAO visual identity. Topo Components is our library of pre-built 
+UI components using [Storybook][5].
 
 ## Contributing
 
 This project is open source and we would love for you to contribute to it. In
-the [CabinDAO  discord server](https://discord.gg/4G6XjsCjM3), there is a channel dedicated to discussing this
-product. Progress, milestones, and on-going initiatives are tracked [here in
-GitHub](https://github.com/CabinDAO/design-system/projects). There is also
-a [discussion board](https://github.com/CabinDAO/design-system/discussions) here
-on GitHub that acts as a good public-interface to the development of this
-product.
+the our [discord server][11], there is a channel dedicated to discussing this 
+product. Progress, milestones, and on-going initiatives are tracked
+[here in GitHub][6]. There is also a [discussion board][7] here on GitHub that 
+acts as a good public interface to the development of this product.
 
-If you would like to contribute, start by reading the [Contribution Guide](./CONTRIBUTING.md) and the [ Code of Conduct ](CONDUCT.md). Open issues
-can be found on this repository.
+If you would like to contribute, start by reading the [Contribution Guide][8] 
+and the [Code of Conduct][9]. Open issues can be found in this repository.
 
 ## Setup
 
-To get the application up and running locally, consult the [Setup Guide](./SETUP.md).
-
-## Cabin
-
-[Cabin](https://www.creatorcabins.com) is a digitally-native organization (often
-called a DAO) building a decentralized city by creators, for creators.  You can
-learn more about us by [visiting our website](https://www.creatorcabins.com) or
-[joining our Discord server](https://discord.gg/4G6XjsCjM3). Please consult our
-[Brand Guidelines](https://github.com/CabinDAO/design-system) on how to work
-within our visual identity. 
+1. Fork the Topo repository into your Github account.
+2. Clone the forked repo to your local system.
+3. Navigate to the newly created project directory.
+4. `yarn` or `npm i` will install the Topo package along with its dependencies.
```

Comment:
technically just installs the package's dependencies.

Reply:
let's add a note on using Storybook to visually verify components

Reply:
Thank you for the clarification regarding package dependencies.

Do I resolve all of the above and then submit a pull request?

Reply:
let's iterate on this one, especially since there appears to be a merge conflict